### PR TITLE
Feature/show all athorization

### PR DESCRIPTION
**Pull request by Dpksolanki**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -36,7 +37,8 @@ function App() {
     redirectBaseUrl,
     embedBaseUrl,
     embedProjectUrl,
-    template
+    template,
+    auth
```
[the remark this answers is not in the document]
Comment (SiDDhartharG) [reply src/App.js]: @Dpksolanki

File: src/App.js [hunk 2]
```diff
@@ -36,7 +37,8 @@ function App() {
     redirectBaseUrl,
     embedBaseUrl,
     embedProjectUrl,
-    template
+    template,
```
Comment (SiDDhartharG) [inline src/App.js:40]: sdfsdffs

**Review by SiDDhartharG — CHANGES_REQUESTED** [review]

File: src/components/authDataTable/authDataTable.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+import './authDataTable.css'
+import { useCustomSelector } from '../../utils/deepCheckSelector'
+import { listOfUsers } from '../../store/orgUsers/orgUsersSelector'
+import selectActiveUser from '../../store/user/userSelector'
+import { getOrgCreator } from '../../store/orgs/orgsSelector'
+
+function AuthData() {
+  // const orgId = 'U8WBg9R2Hkba'
+  // const orgId = '8t6lTca0C7ta'
+  const usersList = useCustomSelector((state) => listOfUsers(state))
```
Comment (SiDDhartharG) [inline src/components/authDataTable/authDataTable.js:15]: update this as per the discursion
we already have user id mapping

**Review by SiDDhartharG — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dpksolanki) [reply src/App.js]: it resolve

File: src/components/authDataTable/authDataTable.js (hunk 3 above)
Comment (Dpksolanki) [reply src/components/authDataTable/authDataTable.js:15]: Done

**Review by SiDDhartharG — APPROVED** [review]

**Review by SiDDhartharG — APPROVED** [review]

**Pull request merged**